rpc/errors.go

```diff
@@ -27,7 +27,7 @@ type methodNotFoundError struct {
 func (e *methodNotFoundError) ErrorCode() int { return -32601 }
 
 func (e *methodNotFoundError) Error() string {
-	return fmt.Sprintf("The method %s%s%s does not exist/is not available", e.service, serviceMethodSeparator, e.method)
+	return fmt.Sprintf("The method %s%s%s does not exist/is not available", e.service, serviceMethodSeparators[0], e.method)
```

Comment:
Instead of wrestling with a new method or adding logic, I'm just opting for simplicity here; this `serviceMethodSeparators` var will never change (it's only a `var` b/c it's a `[]string`), and `rpc.discover` is the only method that uses '.` as a sep.